docs_user/modules/proc_adopting-telemetry-services.adoc

```diff
@@ -99,8 +115,12 @@ sources:
         - volume.size
         - image.size
         - cpu
-        - memory
+        - memory.usage
 EOF
+----
 
+. Update ceilometer configuration with new pollsters:
++
+----
 oc patch secret ceilometer-config-data  --patch="{\"data\": { \"polling.yaml\": \"$(base64 -w0 polling.yaml)\"}}"
```

Comment:
If the user does this, it will be like nothing, as the operator will promptly trigger a reconcile and it will overwrite the user's changes. Configuring pollsters by editing the secret is not possible while the telemetry-operator is running.

Reply:
We dont have one, the telemetry-operator will always apply the original configuration and polling.yaml cannot benefit from the custom configuration module that is used to modify the ceilometer.conf configuration.

Reply:
It needs to be done via the "defaultConfigOverride" attribute of ceilometer, to specify the wanted polling.yaml.j2 file:

```
ceilometer:
        defaultConfigOverwrite:
          polling.yaml.j2: |
            ---
            sources:
              - name: pollsters
                interval: 100
                meters:
                  - volume.*
                  - image.size
        enabled: true
        secret: osp-secret
```